manager/assets/modext/widgets/resource/modx.panel.trash.js

```diff
@@ -41,4 +42,3 @@ MODx.panel.Trash = function (config) {
 };
 Ext.extend(MODx.panel.Trash, MODx.FormPanel);
 Ext.reg('modx-panel-trash', MODx.panel.Trash);
```

Comment:
```suggestion
Ext.reg('modx-panel-trash', MODx.panel.Trash);

```

Тут пустую строку нужно вернуть

Reply:
Она там есть, github не показывает последнюю. А удалил я лишнюю, их было 2, проверь.

Reply:
Если так, то отстал